clustering/polus-hdbscan-clustering-plugin/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM labshare/polus-bfio-util:2.1.9
+COPY VERSION /
+COPY src ${EXEC_DIR}/.
+RUN apt --no-install-recommends -y autoremove --purge python3.9-minimal python3.9\
+    && apt-get update && apt-get install --no-install-recommends -y build-essential python3.9-dev\
+    && pip3 install -r ${EXEC_DIR}/requirements.txt --no-cache-dir
```

Comment:
It's unclear to me why we need to include build-essential and python3.9-dev. The base container is based on Ubuntu, so precompiled wheels should exist for this.

Reply:
No binary wheel for python3.9.